doc/about/about.rst

```diff
@@ -1,72 +1,108 @@
-Frequently Asked Questions
-==========================
```

Comment:
What is the thinking behind merging the FAQ page and the about page?

Reply:
basically I was trying to separate out "content about preparing repositories from Binder" from "information specifically about `mybinder.org`". We already had a [FAQ-like about page](https://mybinder.readthedocs.io/en/latest/about.html) in our community section. In addition, most of the entries in the FAQ were actually about `mybinder.org` instead of building binder repositories, so I moved them over to the `mybinder.org-specific` section. Note that there *is* still a FAQ for the "Using" section, but it has been moved with the re-org: https://mybinder--213.org.readthedocs.build/en/213/using/faq.html